docs/use cases/README.md

```diff
@@ -63,6 +63,39 @@
 
 ### Діаграма прецедентів для учасника проєкту
 
+<div style="
+    border-radius:4px;
+    border: 1px solid #cfd7e6;
+    padding: 1em;">
+
+@startuml
+
+  actor "Учасник проєкту" as ProjectMember #7be37f
+
+  usecase "Управління\nзавданнями" as TaskManagement #abedae
+
+  usecase "Створити\nзавдання" as CreateTask #e8ffe9
+  usecase "Редагувати\nзавдання" as EditTask #e8ffe9
+  usecase "Змінити\nстатус завдання" as ChangeStatus #e8ffe9
+  usecase "Видалити\nзавдання" as DeleteTask #e8ffe9
+  usecase "Коментувати\nзавдання" as CommentTask #e8ffe9
+  usecase "Дивитись\nпрогрес проєкту" as ViewProgress #e8ffe9
+  usecase "Вийти\nз проєкту" as LogOut #e8ffe9
```

Comment:
- remove colors for all use cases except `ProjectMember`, `TaskManagement`, `LogOut`
- `ProjectMember` color should be the same as on general schema, colors for other 2 use cases should be different shades of green, they also should be more light, for example `#e4ffdb` and `#d9fff0`
- there should be 4 spaces instead of 2 in this scheme